app/models/reports/export_raw.rb

```diff
@@ -69,9 +69,9 @@ def default_report_hash
         reservation_start_time: -> (od) { od.reservation.reserve_start_at if od.reservation },
         reservation_end_time: -> (od) { od.reservation.reserve_end_at if od.reservation },
         reservation_minutes: -> (od) { od.reservation.duration_mins if od.reservation },
-        actual_start_time: -> (od) { od.reservation.actual_start_at if od.reservation },
-        actual_end_time: -> (od) { od.reservation.actual_end_at if od.reservation },
-        actual_minutes: -> (od) { od.reservation.actual_duration_mins if od.reservation },
+        actual_start_time: -> (od) { od.time_data.actual_start_at if od.time_data },
```

Comment:
I'm ignoring the rubocop warning here to maintain consistency with the rest of the file. Doing the rubocop cleanup should be its own PR/commit.